deployment/distribution/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright (c) 2018 Codice Foundation
+
+Released under the GNU Lesser General Public License; see
+http://www.gnu.org/licenses/lgpl.html
+*/
+// Build file
+apply(plugin = "application")
+
+description = "Script and CLI to run the tests."
+
+group = "org.codice.samlconf.deployment"
+
+configure<ApplicationPluginConvention> {
+    applicationName = "samlconf"
+    mainClassName = "org.codice.ctk.CommandKt"
+
+    applicationDefaultJvmArgs = listOf("-Dapp.home=SAMLCTK_APP_HOME")
+
+    applicationDistribution.from("src/main/resources/") {
+        into( "conf")
+    }
+
+    // Copy implementation examples into the distribution
+    project(":external:implementations").getSubprojects().forEach { subProject ->
+        applicationDistribution
+                .into("implementations/" +
+                        subProject.name.replace("samlconf-", "")
+                                .replace("-impl", "")) {
+            from (subProject.file("build/resources/main"))
+            from (subProject.file("build/libs"))
+        }
+    }
+}
+
+dependencies {
+    compile(project(":external:implementations:samlconf-ddf-impl"))
+    compile(project(":external:implementations:samlconf-keycloak-impl"))
+    compile(project(":library"))
+    compile(project(":external:samlconf-plugins-api"))
+    compile(project(":ctk:common"))
+    compile(project(":ctk:idp"))
+
+    compile("us.jimschubert:kopper-typed:0.0.3")
+    compile("org.fusesource.jansi:jansi:1.17.1")
+}
+
+// We don"t want to include the implementation jars in the classpath
+// since they"re loaded from a specific directory.
+tasks {
+    "copyToLib"(Copy::class) {
+        from(configurations.runtime) {
+            // TODO RAP 27 Jul 18: Fix this exclusion
```

Comment:
@blen-desta I was hoping you could specifically take a look at this and see if you can help me figure out if we still need it and what it should look like.

Reply:
I would expect this to still be necessary. You should be able to manually verify that by removing the jar from the `implementations/ddf` folder under the exploded distro and running the default options. The tests would still run because the ddf jar is on the classpath, even though it's not being loaded from the implementation folder.

Reply:
@blen-desta Nope. That's not supported. That's why I was hoping for a little help. ;)